xwalk_core_library_android.gypi

```diff
@@ -18,7 +18,6 @@
         'api_files': [
           '<(DEPTH)/xwalk/runtime/android/core/src/org/xwalk/core/JavascriptInterface.java',
           '<(DEPTH)/xwalk/runtime/android/core/src/org/xwalk/core/XWalkActivity.java',
-          '<(DEPTH)/xwalk/runtime/android/core/src/org/xwalk/core/XWalkApplication.java',
```

Comment:
We won't remove XWalkApplication completely. The user can still use it without compilation error, it just do nothing.